.github/workflows/ci-macarm.yml

```diff
@@ -0,0 +1,156 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the OpenColorIO Project.
+#
+# GitHub Actions workflow file
+# https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions
+
+name: CI_MacARM
+
+# This workflow is for CI for ARM-based MacOS. In an ideal world, this would
+# just be another matrix entry in the main CI workflow, but MacOS ARM runners
+# too expensive to run on every PR push. So we break into a separate workflow
+# to have more fine-grained control over when it runs.
+
+# This section controls when the workflow runs.
+on:
+  # Run upon push, but only for tagged versions, or if the branch name
+  # contains the substrings "macarm", "-alpha", "-beta", or is called
+  # "release".
+  push:
+    tags:
+      - v**
+    branches:
+      - '*macarm*'
+      - '*alpha*'
+      - '*beta*'
+      - 'release'
+  # Run for pull requests whose branch name includes "macarm" (this allows
+  # us to test specific PRs that we think need ARM verification).
+  pull_request:
+    tags:
+      - v**
+    branches:
+      - '*macarm*'
+      - '*alpha*'
+      - '*beta*'
+      - 'release'
+
```

Comment:
Maybe we should schedule a job here, as Larry did, to run once a month regardless?

Reply:
OCIO did not have that schedule so I decided to remove it from the OIIO action that I copied over. It makes sense to run it at least once a month so we don't discover issues very late into the release cycle.

Reply:
OIIO runs every month on the 27th a few days before their patch releases. Does OCIO have a regular patch release schedule? Wondering when we should run for OCIO.